.github/scripts/spawn_agent.py

```
@@ -0,0 +1,137 @@
+
+    # Spawn the agent
+    print(f"🤖 Spawning Codegen agent for {issue_type} issue...")
+
+    result = spawn_agent(prompt, metadata={
+        "issue_type": issue_type,
+        "pr_number": pr_number,
+        "commit_sha": commit_sha
+    })
+
+    # Output results
+    print(f"✅ Agent spawned successfully!")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)